application/use_cases/stock_items/create_stock_item/create_stock_item_input_port.py

```diff
@@ -4,6 +4,6 @@
 
 
```

Comment:
It is, just omitted from the domain entity for now for simplicity. I want to keep stock items and products as two separate systems that can be used independently of each other. The Woolies or Coles API may go down or block us some day, so we wouldn't want the whole app being unusable because of that.

Reply:
Also unsure if it would ever be needed on creation of a stock item, maybe, maybe not

Reply:
That's a good idea.
I don't think we will ever need to couple them tbh. 
I don't currently see any benefit to adding that complexity.